src/plots.jl

```diff
@@ -15,53 +51,36 @@ function photon_dens_integrated(sim; species = "photon", direction = :x, plot_ti
               title = plot_title
     )
 
+    nᵧ_int = dropdims(sum(nᵧ*cell_volume(file), dims=dir_to_idx(direction)), dims=dir_to_idx(direction))
+    n₀_int = dropdims(sum(n₀*cell_volume(file), dims=dir_to_idx(direction)), dims=dir_to_idx(direction))
+    plt = Plots.heatmap([1:1:size(nᵧ_int,1)]*a, [1:1:size(nᵧ_int,2)]*a, transpose(nᵧ_int ./ n₀_int); kwargs...)
+    
     if direction == :x
-        nᵧ_int = dropdims(sum(nᵧ*cell_volume(file), dims=1), dims=1)
-        n₀_int = dropdims(sum(n₀*cell_volume(file), dims=1), dims=1)
-        plt = Plots.heatmap([1:1:size(nᵧ_int,1)]*a, [1:1:size(nᵧ_int,2)]*a, transpose(nᵧ_int ./ n₀_int), size = (600,500); kwargs...)
-        
-        length_y = ustrip(u"μm", get_parameter(sim[1], :y_max) - get_parameter(sim[1], :y_min))
-        length_z = ustrip(u"μm", get_parameter(sim[1], :z_max) - get_parameter(sim[1], :z_min))
-
         Plots.plot!(plt,
               xlabel = "y (μm)", 
-              xticks=(1:2:length_y),
+              xticks = (1:2:ustrip(u"μm", domain_length(sim, :y))),
```

Comment:
There should be no need for `ustrip` since we are using `UnitfulRecipes`

Reply:
I've tried without ustrip and I get a compatibility error for dimensions. Maybe because the 1 and 2 in `(1:2:domain_length(...))` are unitless.

Reply:
Yes, they should have units